apps/judicial-system/api/src/app/modules/case-list/models/caseList.model.ts

```diff
@@ -158,4 +158,7 @@ export class CaseListEntry {
   // automatically.
   @Field(() => Boolean, { nullable: true })
   readonly publicProsecutorIsRegisteredInPoliceSystem?: boolean
+
+  @Field(() => Boolean, { nullable: true })
+  readonly isRegisteredInPrisonSystem?: boolean
 }
```

Comment:
Addin this here if fine although it will not be used after we implement the new case table framework.